quarkchain/config.py

```diff
@@ -220,6 +220,7 @@ class QuarkChainConfig(BaseConfig):
 
     PROOF_OF_PROGRESS_BLOCKS = 1
     TESTNET_MASTER_ADDRESS = "199bcc2ebf71a851e388bd926595376a49bdaa329c6485f3"
+    GUARDIAN_PUBLIC_KEY = "00" * 64  # TODO: update this
```

Comment:
huh?

Reply:
we'll put a real key here later to turn on guardian